### PR TITLE
RHODS-12410 - Create a Robot framework test for running DW tests

### PR DESCRIPTION
**Pull request by jiripetrlik**
The goal is to allow tests from Distributed Workloads repo to be run as robot framework tests (https://github.com/opendatahub-io/distributed-workloads/tree/main/tests).

### CODE REVIEW

File: ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        FAIL    Unable to clone DW repo ${DW_REPO_URL}:${DW_REPO_BRANCH}:${DW_DIR}
+    END
+
+Run Distributed Workloads Tests
+    [Documentation]   Run distributed workloads E2E tests from integration directory
+    [Arguments]    ${DW_DIR}    ${DW_TEST_RESULT_FILE}
+    ${result} =    Run Process
+    ...    go test -v -timeout ${GO_TEST_TIMEOUT} ./integration 2>&1 > ${DW_TEST_RESULT_FILE}
+    ...    shell=true    cwd=${DW_DIR}/tests    env:ODH_NAMESPACE=redhat-ods-applications
+    ...    env:DW_TEST_TIMEOUT_LONG=${DW_TEST_TIMEOUT_LONG}
```
Comment (sutaakar) [inline ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:31]: Where is the `DW_TEST_TIMEOUT_LONG` declared and what does it represent? In DW tests we use env variable `CODEFLARE_TEST_TIMEOUT_LONG` - https://github.com/project-codeflare/codeflare-common/blob/main/support/support.go#L54

Comment (jiripetrlik) [reply ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource]: @sutaakar Thank you for noticing it! It should be fixed now.

**Comment by jiripetrlik** [comment]
@aloganat @manosnoam  Hello, will you have a time to review this PR?

**Review by sutaakar — COMMENTED** [review]
Looks good to me

**Review by aloganat — APPROVED** [review]
lgtm, as the DW tests passed in the sanity run execution

**Review by sutaakar — APPROVED** [review]

**Pull request merged**